packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -71,21 +70,21 @@ module.exports = function (
               const password = new Password(
                 oldAuthPW,
                 emailRecord.authSalt,
-                emailRecord.verifierVersion,
+                emailRecord.verifierVersion
```

Comment:
@vbudhram I think our prettier configs must be in conflict somehow on trailing commas. 😬